src/DynamoCoreWpf/ViewModels/Core/PortViewModel.cs

```diff
@@ -641,19 +641,19 @@ private void RefreshPortColors()
             {
                 if (_port.IsConnected)
                 {
-                    PortBackgroundColor = (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["PortIsConnectedBackground"];
-                    PortBorderBrushColor = (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["PortIsConnectedBorderBrush"];
+                    PortBackgroundColor = new SolidColorBrush(Color.FromRgb(70, 90, 99));
+                    PortBorderBrushColor = new SolidColorBrush(Color.FromRgb(106, 192, 231));
                 }
                 else
                 {
-                    PortBackgroundColor = (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["PortDefaultBackground"];
-                    PortBorderBrushColor = (SolidColorBrush)SharedDictionaryManager.DynamoColorsAndBrushesDictionary["PortDefaultBorderBrush"];
+                    PortBackgroundColor = new SolidColorBrush(Colors.Transparent);
+                    PortBorderBrushColor = new SolidColorBrush(Color.FromRgb(204, 204, 204));
```

Comment:
Ugh 😨

Reply:
yeah... agreed, we should figure out why this does not work during testing instead of making the code less maintainable.

If you're blocked @OliverEGreen I think @QilongTang can file a task to have the teams investigate - but a minimal repro case will be needed.

Reply:
Filed as https://jira.autodesk.com/browse/DYN-3919